snippets/Check_for_palindrome.md

```diff
@@ -0,0 +1,10 @@
+### Check For Palindrome
+
+```
+function palindrome(str) {
```

Comment:
Use ES6 syntax `palidrome = str =>` to fit with the rest of the snippets.

Reply:
Did perfect cloning for es6